src/main/java/com/taskadapter/redmineapi/RedmineManagerFactory.java

```diff
@@ -209,6 +209,22 @@ public static ClientConnectionManager createConnectionManagerWithExtraTrust(Coll
     	return createConnectionManager(BetterSSLFactory.createSocketFactory(trustStores));
     }
 
+        /**
+     * Creates a connection manager with extended trust relations and Client 
+     * Certificate authentication. It would use both default system trusted 
+     * certificates as well as all certificates defined in the 
+     * <code>trustStores</code>.
+     * @param keyStore key store containing the client certificate to use.
+     * @param keyStorePassword the keyStore password string.
+     * @param trustStores list of additional trust stores.
```

Comment:
it is worth documenting what this list is for

Reply:
That's the same list you use for the extra CAs (It's basically an extension of the function that was already there, but I'll add more comprehensive comments there

Reply:
Added extra comments, and the JavaDoc for the new function that I had missed.